composer.json

```diff
@@ -1,4 +1,12 @@
 {
```

Comment:
@sharprakeshverma You should also add the autoload for test and phpunit as dependence.